src/components/views/dialogs/ExportDialog.tsx

```diff
@@ -163,8 +164,9 @@ const ExportDialog: React.FC<IProps> = ({ room, onFinished }) => {
                     return allowEmpty || !!value;
                 },
                 invalid: () => {
+                    const config = SdkConfig.get();
```

Comment:
Other places, this is wrapped with some sort of `SnakedObject` thing that seems to worry about camelCase backward compatibility. That need seems to predate this change, so I left it out. (Here and below.)